### PR TITLE
Focus visible for tooltip informations in labels

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

Before:
Capture d’écran 2024-05-21 à 14 32 31

After: 
Capture d’écran 2024-05-21 à 14 32 10

-----

I'm also adding an alternative text to the icon, but until I can add a translation mechanism, I'm sticking with a very generic text « ? » for « help ».

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2803/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17267/ui-diff)

File: stories/qa/forms/checkbox.stories.html [hunk 1]
```diff
@@ -1,33 +1,59 @@
...
+			Label<sup class="formLabel-required" aria-hidden="true">*</sup
+			><span class="formLabel-info"><span aria-hidden="true" class="lucca-icon icon-signHelp"></span><span class="u-mask">?</span></span>
 		</label>
 		<span class="checkboxField">
-			<input type="checkbox" class="checkboxField-input" id="CB" aria-labelledby="CB-label" aria-describedby="CB-message" aria-required="true" />
+			<input
+				type="checkbox"
+				class="checkboxField-input"
+				id="CB"
+				aria-labelledby="CB-label"
```
Comment (fbasmaison-lucca) [inline stories/qa/forms/checkbox.stories.html:12]: Is it normal no tag has an `id="*-label"`?

**Review by fbasmaison-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2803/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17281/ui-diff)

**Review by Supamiu — APPROVED** [review]

**Review by fbasmaison-lucca — APPROVED** [review]

**Review by shajz — APPROVED** [review]

**Pull request merged**